### ISSUE TITLE
footnotes_wysiwyg: Improve dialog fields

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-contrib/footnotes#20]
A few suggestions to improve the *Insert footnote* dialog fields:

(a) Footnote text

- The input has a `maxlength` of 128 characters but there are quite long footnotes in the wild, so we should allow 'maxlength' to be higher, maybe 256.
- The size of the input could also be bigger. In my opinion room for two text lines instead of one would be fine for most use cases.

(b) Footnote value

- The label "Footnote value" is not very clear, and the "Footnote" part is repetitive. Let's change it to "Label value" which is slightly more convenient in my opinion (but see also #19).
- I'd suggest to make the input size smaller, e.g. `size="10"` because it's only needed for edge cases,  and in these cases people want to enter only one or a few strings.

With the combination of a bigger input for (a) and a smaller input for (b) we would get a better balanced user interface, depending of the importance of the respective elements.

**Comment by olafgrabienski** [comment]
Here's a screenshot comparison, (1) the current dialog, (2) a mockup of the suggestions:

![screen-backdrop-footnotes-wysiwyg-dialog-fields-current](https://user-images.githubusercontent.com/14188307/46143747-36ae3880-c25b-11e8-9124-5352a7486bf9.png)

![screen-backdrop-footnotes-wysiwyg-dialog-fields-mockup](https://user-images.githubusercontent.com/14188307/46143748-3746cf00-c25b-11e8-96a8-b93573d7e137.png)

**Comment by herbdool** [comment]
I agree, this makes more sense. Perhaps also a description? And maybe just call it "Label"

**Comment by olafgrabienski** [comment]
> Perhaps also a description?

Yes, or is it a help text? Anyway, what do you think about my text suggestion in [issue 19, Add help text to value field](https://github.com/olafgrabienski/footnotes/issues/19#issuecomment-425047816)?

> And maybe just call it "Label"

Good idea, in my opinion, without "value".

Btw, is "footnote label" the common term for a footnote number or symbol? I know the German term ("Fußnotenzeichen") but not the English one. Here's a list of terms I've found searching for a translation of the German term:

- footnote icon
- footnote mark
- footnote marker
- footnote reference mark
- footnote sign

Is one of these more convenient than "footnote label"?

**Comment by herbdool** [comment]
Good question. I can't think of a word for it. Of those options maybe "marker" is the closest.

**Comment by olafgrabienski** [comment]
Thanks for the feedback. Agreed, "marker" sounds good in this context!

**Comment by olafgrabienski** [comment]
> The label "Footnote value" is not very clear, and the "Footnote" part is repetitive.

In contrast to my first thoughts I'm considering to **not** drop the "Footnote" part of the label even if it's repetitive. Without the "Footnote" part a term like "Marker" has no context, which is fine in the *Insert footnote* dialog but not in other places. It would e.g. be quite difficult to translate the term.

**Comment by olafgrabienski** [comment]
New proposal - based on the comments above and issue #19 about help texts (I've closed that issue in favor of this one):

**Footnote text:**

- bigger field => question: would it still be a text field or better a textarea?
- max-length at least 256

**Footnote marker:**

- smaller field (size=14 now)

Help text or description:
> Leave empty to use automatic numbering.
> To start numbering from a specific value, enter a number.
> To use a footnote symbol, enter your symbol.

---

**Mockup:**

![screen-backdrop-footnotes-wysiwyg-dialog-fields-mockup-02](https://user-images.githubusercontent.com/14188307/46210714-cd4d2900-c331-11e8-83e4-1b197b2a9b59.png)
